modules/scylladb/src/main/java/org/testcontainers/containers/ScyllaDBContainer.java

```diff
@@ -0,0 +1,194 @@
+package org.testcontainers.containers;
+
+import com.github.dockerjava.api.command.InspectContainerResponse;
+import org.apache.commons.io.IOUtils;
+import org.testcontainers.containers.delegate.ScyllaDBDatabaseDelegate;
+import org.testcontainers.delegate.DatabaseDelegate;
+import org.testcontainers.ext.ScriptUtils;
+import org.testcontainers.ext.ScriptUtils.ScriptLoadException;
+import org.testcontainers.utility.DockerImageName;
+import org.testcontainers.utility.MountableFile;
+
+import java.io.IOException;
+import java.net.InetSocketAddress;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.util.Optional;
+
+import javax.script.ScriptException;
+
+/**
+ * Testcontainers implementation for ScyllaDB.
+ * <p>
+ * Supported image: {@code scylladb}
+ * <p>
+ * Exposed ports: 9042
+ */
+public class ScyllaDBContainer<SELF extends ScyllaDBContainer<SELF>> extends GenericContainer<SELF> {
+
+    private static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse("scylladb/scylla:5.2.9");
+
+    public static final Integer CQL_PORT = 9042;
+
+    private static final String DEFAULT_LOCAL_DATACENTER = "datacenter1";
+
+    private static final String CONTAINER_CONFIG_LOCATION = "/etc/scylla";
+
+    private static final String USERNAME = "scylladb";
+
+    private static final String PASSWORD = "scylladb";
+
+    private String configLocation;
+
+    private String initScriptPath;
+
+    private boolean enableJmxReporting;
+
+    public ScyllaDBContainer(String dockerImageName) {
+        this(DockerImageName.parse(dockerImageName));
+    }
+
+    public ScyllaDBContainer(DockerImageName dockerImageName) {
+        super(dockerImageName);
+        dockerImageName.assertCompatibleWith(DEFAULT_IMAGE_NAME);
+
+        addExposedPort(CQL_PORT);
+        this.enableJmxReporting = false;
+
+        withEnv("CASSANDRA_SNITCH", "GossipingPropertyFileSnitch");
+        withEnv("JVM_OPTS", "-Dcassandra.skip_wait_for_gossip_to_settle=0 -Dcassandra.initial_token=0");
```

Comment:
those are not relevant to scylla, you need commandline option for `skip_wait_for_gossip_to_settle`